code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -465,7 +465,7 @@
 	desc = "A colorful, sugary gumball."
 	icon = 'icons/obj/lollipop.dmi'
 	icon_state = "gumball"
-	list_reagents = list("sugar" = 5, "bicaridine" = 2, "kelotane" = 2)	//Kek
+	list_reagents = list("sugar" = 5)
```

Comment:
Not too sure about replacing these with something else? Salglu perhaps?

Reply:
what did these old chems do before?

Reply:
It is my educated opinion that this barely matters. Add 1u of salglu and we can merge this baby.